combat-trainer.lic

```diff
@@ -1181,8 +1184,14 @@ class SpellProcess
     return unless DRStats.necromancer?
     return unless @necromancer_healing
     return if game_state.casting
-    return unless game_state.prepare_consume
 
+    if health <= @siphon_vit_threshold && @necromancer_healing['Siphon Vitality'] && game_state.npcs?
```

Comment:
I don't think `game_state.npcs?` will work since the function is named `npcs`. Maybe `game_state.npcs.empty?`